source/reference/data-types.txt

```diff
@@ -285,13 +287,13 @@ returned:
      - **{ "_id": 1, "val": NumberDecimal( "9.99" ), "description": "Decimal" }**
 
    * - **{ val: 10 }**
-     - | **{ "_id": 3, "val": 10, "description": "Double" }**
-       | **{ "_id": 4, "val": NumberLong(10), "description": "Long" }**
+     - | **{ "_id": 3, "val": 10, "description": "Int" }**
+       | **{ "_id": 4, "val": 10, "description": "Long" }**
        | **{ "_id": 5, "val": NumberDecimal( "10.0" ), "description": "Decimal" }**
 
    * - **{ val: NumberDecimal( "10" ) }**
-     - | **{ "_id": 3, "val": 10, "description": "Double" }**
-       | **{ "_id": 4, "val": NumberLong(10), "description": "Long" }**
+     - | **{ "_id": 3, "val": 10, "description": "Int" }**
+       | **{ "_id": 4, "val": 10, "description": "Long" }**
```

Comment:
This was surprising, thanks for making us notice :) I’ve opened https://jira.mongodb.org/browse/MONGOSH-809 and we’ve decided that we’ll change this so that the output here should be `Long(10)` instead of just `10`.

Reply:
Cool! Glad to help. 

Should I wait for the fix to go in or go with the current output and update later?